overrides/scripts/Midgame.zs

```diff
@@ -4,6 +4,14 @@ import crafttweaker.item.IIngredient;
 import crafttweaker.data.IData;
 import scripts.CommonVars.makeShaped as makeShaped;
 
+//EV Electrolyzer
+<gregtech:machine:243>.displayName = "Advanced Electrolyzer III";
+<gregtech:machine:243>.addTooltip("Molecular Disintegrator E-4904");
+
+//EV Air Collector
+<gregtech:machine:980>.displayName = "Advanced Air Collector III";
+<gregtech:machine:980>.addTooltip("Atmosphere Collector III");
+

```

Comment:
I think we should hold off on trying to fix the machine names for now as the SoG renaming not taking effect should be investigated further by us in SoG before we resort to a kludge with CraftTweaker.

As an aside, the "funny name" added to the tooltip ends up at the bottom which is inconsistent with SoG sticking them at the top of the tooltip.